*** tests/A-21-13-M-ok.out	2024-10-25 03:49:31.591699182 +0100
--- tests/A-21-13-M-ok.outhyp	2024-10-25 03:49:58.926635288 +0100
***************
*** 16,30 ****
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Identificador único do animal: A vacina 'Vac1' não é apropiada para o animal 'AA3'.
! Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Identificador único do animal: A vacina 'Vac2' não é apropiada para o animal 'AA1'.
! Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
--- 16,28 ----
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Identificador único do animal: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Identificador único do animal: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
