*** testsBoth/9-remove-add-zookeeper-responsibility.out	2024-10-25 21:51:06.000000000 +0100
--- testsBoth/9-remove-add-zookeeper-responsibility.outhyp	2024-10-25 22:11:37.578025876 +0100
***************
*** 135,141 ****
  10 - Consultas
  0 - Sair
  Escolha uma opção: 5
! 1428
  Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
--- 135,141 ----
  10 - Consultas
  0 - Sair
  Escolha uma opção: 5
! 1425
  Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
